pkg/apis/clusters/v1beta1/types.go

```diff
@@ -231,6 +231,14 @@ type ManagedClusterStatus struct {
 	// +optional
 	APIServerURL string `json:"apiserverURL,omitempty"`
 
+	// APIServerURLOutCls indicates the advertising url/address of managed Kubernetes cluster, out-cluster
+	// +optional
+	APIServerURLOutCls string `json:"apiserverURLOutCls,omitempty"`
+
+	// APIServerConfig indicates the advertising config of managed Kubernetes cluster
```

Comment:
No need. There are already a secret in the dedicated namespace, which can be used as the kubeconfig for child clusters.

And it is NOT safe to put `kubeconfig` in `ManagedCluster`.